common_app/io.c

```diff
@@ -0,0 +1,162 @@
+/*****************************************************************************
+ *   Ledger App Boilerplate.
```

Comment:
It should not be linked to Boilerplate anymore :)
And this file could be updated now that we have merged a new version in app-boilerplate adding the support to Ledger Stax!